.vib/jenkins/cypress/cypress/integration/jenkins_spec.js

```diff
@@ -0,0 +1,94 @@
+/// <reference types="cypress" />
+import { random, checkErrors } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('.alert-danger').should('not.exist');
+  cy.get('div#tasks');
+
+  cy.get('[href$="/logout"]').click();
+  cy.get('form[name*="login"]');
+});
+
+it('should be possible to create a new Git Project', () => {
+  cy.login();
+  cy.visit('/view/All/newJob');
+
+  cy.fixture('items').then((item) => {
+    cy.get('#name').type(`${item.freestyleProject.name}-${random}`);
+    cy.contains(item.freestyleProject.type).click();
+    cy.contains('button', 'OK').should('be.enabled').click();
+    cy.contains('Error').should('not.be.visible');
```

Comment:
There is a hidden Error element in the DOM that forced me to go with `not.be.visible` instead of `not.exist`:

![Screenshot 2022-05-16 at 12 18 39](https://user-images.githubusercontent.com/24253844/168571970-a3c87244-0532-4d8c-91d8-6260ad8b6ae7.png)